.pylintrc

```diff
@@ -515,7 +520,7 @@ known-third-party=enchant
 max-args=10
 
 # Maximum number of attributes for a class (see R0902).
-max-attributes=7
+max-attributes=9
```

Comment:
Why change it to 9 or just why not remove it?

Reply:
i am going the conservative way here :-)